scripts/prepare_netplugin_images.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
This feels wrong. Can you explain why this approach is needed?

Reply:
we need a way to build images that not on dockerhub